themes/theme-b2b/src/components/Message.styles.ts

```diff
@@ -16,15 +16,15 @@ export const Message: ThemeComponent<'Message'> = {
       },
     }
   ),
-  content: cva('pt-1 leading-[1.125]'),
-  icon: cva('flex h-4 w-4', {
+  title: cva('font-bold'),
+  content: cva(''),
+  icon: cva('', {
     variants: {
       variant: {
-        warning: 'fill-fill-warning',
-        info: 'fill-fill-info',
-        error: 'fill-fill-error',
+        warning: 'text-fill-warning',
```

Comment:
es gibt extra fill colors

Reply:
wenn du das jetzt änderst muss jemand sie auch in figma ändern

Reply:
oder nicht

Reply:
fixed :)